src/Mumble.proto

```diff
@@ -20,6 +20,35 @@ message Version {
 	optional string os_version = 4;
 }
 
+// Send after the version packet, indicate the protocol preference of the client.
+// - This message is not required for text-only clients. It can be send before or after
+//   the Authenticate message. However, it has be sent prior to a client attempts to send
+//   or receive any audio data.
+// - As reply, the server will send an Capabilities message as well, with only one item in
+//   the list, indicating the protocol it determines to use.
```

Comment:
This should be the comment for the actual protocol field as (again) this message will be used for other stuff as well.